multiapps-controller-process/src/test/java/org/cloudfoundry/multiapps/controller/process/steps/UploadAppStepTest.java

```diff
@@ -49,8 +50,9 @@ void testGetTimeoutWithAppParameter() {
         private void testGetTimeout(CloudApplicationExtended app, int expectedUploadTimeout) {
             context.setVariable(Variables.APP_TO_PROCESS, app);
 
-            int uploadTimeout = step.getTimeout(context);
-            assertEquals(expectedUploadTimeout, uploadTimeout);
+            var expectedTimeout = Duration.ofSeconds(expectedUploadTimeout);
+            var actualTimeout = step.getTimeout(context);
```

Comment:
I've seen that Sonar forces us to use var but do we really want it?

Reply:
Maybe not in every instance Sonar suggests but I think this is appropriate.